styles/customSheet.css

```diff
@@ -0,0 +1,262 @@
+    width: 4.8rem;
+    height: 4.8rem;
+    border-radius: 0.8rem;
+}
+
+.menu-container--menu:hover {
+    cursor: pointer;
+}
+
+.display_data--text {
+    width: 100%;
+    font-weight: 400;
+    font-size: 1.4rem;
+    color: hsl(243, 100%, 93%);
+    text-align: center;
+}
+
+.display_data--text span {
+    font-weight: 700;
+}
+
+.progress {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 1.2rem;
+}
+
+.slider--outer {
+    width: 100%;
+    border-radius: 5rem;
+    height: 2rem;
+    padding: 3px;
+    background-color: hsl(229, 57%, 11%);
+}
+
+.slider {
+    position: relative;
+    width: 81.5%;
+    height: 100%;
+    padding: 2px;
+    border-radius: 5rem;
+    background: linear-gradient(90deg,hsl(6, 100%, 80%), hsl(335, 100%, 65%));
+}
+
+.toggle {
+    background-color: white;
+    width: 1rem;
+    height: 1rem;
+    border-radius: 50%;
+    position: absolute;
+    right: 0.25rem;
+}
+
+.progressMark {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    color: hsl(243, 100%, 93%);
+    font-weight: 700;
+    font-size: 1.2rem;
+}
+
+.data-left {
+    position: absolute;
+    width: 18rem;
+    height: 7.2rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: white;
+    gap: 1rem;
+    border-radius: 0.8rem;
+    bottom: 0;
+    right: 50%;
+    transform: translate(50%, 50%);
+
+}
+
+.text {
+    font-size: 4rem;
+    color: hsl(229, 57%, 11%);
+}
+
+.text, .text-small {
+    font-weight: 700;
+}
+
+
+.text-small{
+    font-size: 1.2rem;
+    letter-spacing: 0.1rem;
+    color: hsl(229, 7%, 55%);
+}
+
+.pointing-arrow {
+    position: inherit;
+    display: none;
+    width: 2.2rem;
+    height: 2.5rem;
+    clip-path: polygon(100% 0, 0 0, 100% 100%);
+    background-color: inherit;
+    bottom: 3px;
+    right: 0;
+    transform: translate(0,100%);
+}
+
+.attribution {
+    transform: translateY(7.5rem);
+    font-size: 12px; 
+    text-align: center; 
+    color: hsl(229, 7%, 55%);
+    font-weight: 400;
+}
+
+.attribution a { 
+    color: white;
+    text-decoration: none;
+    font-weight: 700; 
+}
+
+@media (min-width: 1024px){ 
+    body {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        min-height: 100%;
+        margin: auto;
+        background: hsla(229, 57%, 11%, 1) url('../images/bg-desktop.png');
+        background-repeat: no-repeat;
+        background-size: 100% auto;
+        background-position: bottom;
+    }
+
+    .main {
+        width: max-content;
+        flex-direction: row;
+        justify-content: center;
+        align-items: flex-start;
+        gap: 3rem;
+    }
+
+    /* .main-fylo {
+        width: 88%;
+        margin: auto;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        background-color: hsl(228, 56%, 26%);
+    } */
```

Comment:
Please also remove this from your stylesheet. Keep in mind always avoid bad comments. Use good comments for guiding purposes.